test/unit/definitions/divorce/AuthorisationCaseEvent.test.js

```diff
@@ -1,17 +1,34 @@
 const expect = require('chai').expect;
 const { uniqWith } = require('lodash');
 
-const caseTypeTab = Object.assign(require('definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent'), {});
+const load = require;
```

Comment:
this is needed to avoid:

```
$ eslint --fix .

/Users/patrykj/divorce/div-ccd-definitions/test/unit/definitions/divorce/AuthorisationCaseEvent.test.js
  6:26  error  Unexpected require()  global-require
  7:34  error  Unexpected require()  global-require

✖ 2 problems (2 errors, 0 warnings)

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

If you know any better way to fix it, please let me know.